Protocol.md

```diff
@@ -1,21 +1,24 @@
 ## Status
 
-This protocol is currently a draft for the final specifications. 
-Current version of the protocol is __0.2__ (Major Version: 0, Minor Version: 2). This is currently considered a 1.0 Release Candidate. Final testing is being done in Java and C++ implementations with goal to release 1.0 in the near future. 
+This protocol is currently a __draft__ for the final specifications. 
+Current version of the protocol is __0.2__ (Major Version: __0__, Minor Version: __2__). This is currently considered a 1.0 Release Candidate. 
+
+Final testing is being done in Java and C++ implementations with goal to release 1.0 in the near future. 
```

Comment:
Let's remove this dated line altogether.

Reply:
Should I also remove the `Status` Heading ?